auction-app-frontend/src/services/index.js

```diff
@@ -0,0 +1,2 @@
+export * from "./ProductService";
```

Comment:
Shouldn't thise file be export * from './productService` 
Take care about this, export/import has to be written correctly